ibm/service/vpc/data_source_ibm_is_backup_policy_plans_test.go

```diff
@@ -45,6 +45,47 @@ func TestAccIBMIsBackupPolicyPlansDataSourceBasic(t *testing.T) {
 					resource.TestCheckResourceAttrSet("data.ibm_is_backup_policy_plans.is_backup_policy_plans", "plans.0.lifecycle_state"),
 					resource.TestCheckResourceAttrSet("data.ibm_is_backup_policy_plans.is_backup_policy_plans", "plans.0.name"),
 					resource.TestCheckResourceAttrSet("data.ibm_is_backup_policy_plans.is_backup_policy_plans", "plans.0.resource_type"),
+					resource.TestCheckResourceAttrSet("data.ibm_is_backup_policy_plans.is_backup_policy_plans", "plans.0.remote_copy_policies.#"),
```

Comment:
Remove this here as it might be null and as you have already added test case below